src/widgets/Modal/Modal.tsx

```diff
@@ -1,13 +1,16 @@
 import React from "react";
 import styled from "styled-components";
 import Heading from "../../components/Heading/Heading";
-import Button from "../../components/Button/Button";
 import Flex from "../../components/Flex/Flex";
-import { CloseIcon } from "../../components/Svg";
+import { ArrowBackIcon, CloseIcon } from "../../components/Svg";
+import { IconButton } from "../../components/Button";
 import { InjectedProps } from "./types";
 
 interface Props extends InjectedProps {
   title: string;
+  hideCloseButton?: boolean;
+  onBack?: () => void;
+  bodyPadding?: string;
```

Comment:
Okay, you're probably wondering why I am so lazy. My _ideal_ solution would be to have a `<ModalBody />` component but that means we would have change all existing modals.